ecosystem.config.js

```diff
@@ -16,7 +16,8 @@ function generateApp (env, port) {
     log_date_format: 'MM-DD HH:mm:ss',
     env: {
       NODE_ENV: 'production',
-      PORT: port
+      PORT: port,
+      NET_ENV: env,
```

Comment:
We can use `NODE_CONFIG_ENV` here， avoiding coming up with a new variable for every repo.
https://github.com/node-config/node-config/wiki/Environment-Variables#node_config_env